### PR TITLE
Issue#1

### LINKED ISSUE #1: Move test into serial_coms.py
Move the test code from `serial_coms_test.py` script into the main `serial_coms.py` to be run if the script is `__main__`, then remove `serial_coms_test.py`.

### PR DESCRIPTION
**Pull request by viktoriussuwandi**
fixes #1 

Hi [savob](https://github.com/savob),
just trying to fix on [issue#1](https://github.com/hpvdt/ant_to_serial/issues/1), please check it out

### CODE REVIEW

**Comment by viktoriussuwandi** [comment]
Hi [savob](https://github.com/savob),
Is there anything else needed to merge this PR ?
Please let me now

File: serial_coms.py [hunk 1]
```diff
@@ -110,8 +110,9 @@ def testLine(self, testChar):
         
 if __name__ == "__main__":
     # Tests serial by bouncing a random number through the arduino
+    microController = serialLine(38400) # Sets up serial communication line to STM32 (9600 baud)
+    microController.sendDataBytes(100, 100, 300, 100, 100, 100)
     serialTestLine = serialLine(57600)
```
Comment (savob) [inline serial_coms.py]: Cannot make a second `serialLine` object, this will cause errors on our hardware

File: serial_coms.py [hunk 2]
```diff
@@ -110,8 +110,9 @@ def testLine(self, testChar):
...
-
     serialTestLine.testLine('z')
```
Comment (savob) [inline serial_coms.py]: Change `serialTestLine` to `microController` as to use the one and only `serialLine` object

**Review by savob — CHANGES_REQUESTED** [review]
Thank you, but there are a couple issues I've commented on directly regarding a a second `serialLine` object instance that must be addressed first.

I must also ask, how did you find this repository and why are you helping us?

**Comment by viktoriussuwandi** [comment]
> Thank you, but there are a couple issues I've commented on directly regarding a a second `serialLine` object instance that must be addressed first.
> 
> I must also ask, how did you find this repository and why are you helping us?

Hi @savob,
I have tried to fix as you mentioned above, please let me know if anything need to do to merge this PR.
Anyway, I found this issue last week (posted by you), and interesting to contribute on this good first issue.

Appreciate your kindly response

**Comment by savob** [comment]
I see, so that's how you found us. Do you have a background in embedded software? 

You've still got the issue where a second `serialLine` is initiated. We can only have one initiated on a Raspberry Pi (and thus per script) at a time since there is only one physical serial line on the microcomputer for them to control. We could close it and then reopen it but that is not needed for our application since we assume that we are going to have a constant stream of data.

Please remove the second instantiation of `serialLine` to `microController` entirely on line 115.

File: serial_coms.py [hunk 3]
```diff
@@ -110,8 +110,9 @@ def testLine(self, testChar):
         
 if __name__ == "__main__":
     # Tests serial by bouncing a random number through the arduino
-    serialTestLine = serialLine(57600)
-
-    serialTestLine.testLine('z')
+    microController = serialLine(38400) # Sets up serial communication line to STM32 (9600 baud)
+    microController.sendDataBytes(100, 100, 300, 100, 100, 100)
+    microController = serialLine(57600)
```
Comment (savob) [inline serial_coms.py]: This redundant initialization of `serialLine` will cause issues, please remove this line

**Comment by viktoriussuwandi** [comment]
> I see, so that's how you found us. Do you have a background in embedded software?
> 
> You've still got the issue where a second `serialLine` is initiated. We can only have one initiated on a Raspberry Pi (and thus per script) at a time since there is only one physical serial line on the microcomputer for them to control. We could close it and then reopen it but that is not needed for our application since we assume that we are going to have a constant stream of data.
> 
> Please remove the second instantiation of `serialLine` to `microController` entirely on line 115.

Hopefully meet your requirement this time. But, just let me know if anything still going wrong

Actually, I have embedded some software (libraries) before,  but still newcomer on Raspberry Pi.

**Review by savob — APPROVED** [review]
Alrighty, it looks all good to me now. Thanks for the changes @viktoriussuwandi

**Comment by viktoriussuwandi** [comment]
Thanks for your guidance  @savob

**Pull request merged**